descriptors/ModuleDescriptor-template.json

```diff
@@ -18,9 +18,12 @@
           "methods": ["POST"],
           "pathPattern": "/invoice/invoices",
           "permissionsRequired": ["invoice.invoices.item.post"],
+          "permissionsDesired": ["invoice.acquisitions-units-assignments.item.post"],
```

Comment:
I think better to use common naming patter for such permissions like orders app has (`orders.acquisitions-units-assignments.assign`)